src/components/Table/Table.jsx

```diff
@@ -72,13 +86,27 @@ export function Table({
         dataCy={dataCy}
         {...rest}
       >
+        {withColumnChooser ? (
+          <HeaderUI>
```

Comment:
For MVP we won't have the column chooser but we will still have a header with the View name. [relevant spec](https://paper.dropbox.com/doc/Spec-List-View--BRCgjs8z4tVOOsjbO2y_dKhDAg-H9rEIGV717YlfL8JFCxoW#:uid=700259760955942417510932&h2=Folder-Header).
 Can we tweak this so that whether we use columns or not, we can see the view name in the Header? Feels like we'd need to check for the presence of `withHeader` first and then also `withColumnChooser`, and in MVP only `withHeader` would be true.

If it's useful, in my current branch I made the view name an `<H1/>` with the following css (may need to be modified in your branch)
```
  h1 {
    height: 60px;
    font-size: 18px;
    font-weight: 400;
    line-height: 23.04px;
    padding-top: 16px;
    padding-bottom: 19px;
    padding-left: 15px;
    color: ${getColor('charcoal.800')};
  }
```

Reply:
I added a `headerContent` prop, you can pass any valid element and it will render it on the left inside the `<header>` element of the table (along with column chooser on the left, if enabled). I added your h1 styles as an example on the ConvoList demo